### PR TITLE
Nuka Cola Tweak

### PR DESCRIPTION
**Pull request by ProtozoaPrime**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Added Nuka Cola as a bottled drink
tweak: Replaced drinking glass Nuka Cola with bottled Nuka Cola in vending machines
/:cl:

## About The Pull Request
This pull request adds a bottled version of the Nuka Cola drink to the game, for the express usage of stock in the vending machines. This contains the same amount of reagent as the glass version, and the glass version can still be prepared as normal. Also, when smashing the bottle, you get the cool smashed bottle sprite instead of a virgin smashed drinking glass.

## Why It's Good For The Game
It's a simple change that makes putting Nuka Cola in vending machines work properly, instead of just chucking in a drinking glass

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -326,6 +326,12 @@
 	icon_state = "fernetbottle"
 	list_reagents = list(/datum/reagent/consumable/ethanol/fernet = 100)
 
+/obj/item/reagent_containers/food/drinks/bottle/nukacola
```
Comment (yoyobatty) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:330]: Put comments preceding and following indicating this is a hippie edit

File: code/modules/vending/cola.dm [hunk 2]
```diff
@@ -15,7 +15,7 @@
 					/obj/item/reagent_containers/glass/beaker/waterbottle = 10)
 	contraband = list(/obj/item/reagent_containers/food/drinks/soda_cans/thirteenloko = 6,
 		              /obj/item/reagent_containers/food/drinks/soda_cans/shamblers = 6)
-	premium = list(/obj/item/reagent_containers/food/drinks/drinkingglass/filled/nuka_cola = 1,
+	premium = list(/obj/item/reagent_containers/food/drinks/bottle/nukacola = 1,
```
Comment (yoyobatty) [inline code/modules/vending/cola.dm:19]: Ditto

**Comment by ghost** [comment]
https://github.com/HippieStation/HippieStation/tree/master/hippiestation#please-mark-your-changes
Comments should be standardised like so.

**Review by ghost — APPROVED** [review]

**Comment by AnthonyJNgo** [comment]
+3, this will hopefully get more admin votes in the next day or so then be merged.

**Comment by ghost** [comment]
+1

**Comment by carlopault** [comment]
+1 Nuka Cola good taste in ear

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**